### ISSUE TITLE
Prolonged axillary pressure

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11042]
**Preferred term label:**
Prolonged axillary pressure

**Synonyms**

**Definition (free text, please give PubMed ID)**
History of prolonged compressive pressure in the axillary region

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Compressive radial mononeuropathy

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
according to this, it is less history of than an acute insult: https://www.ncbi.nlm.nih.gov/books/NBK539895/
I am trying to find examples of chronic compression injuries but could not...

**Comment by MickeySegal** [comment]
One can quibble over what counts as acute versus prolonged, but it is referred to as prolonged:
https://www.ncbi.nlm.nih.gov/books/NBK557520/
"Compressive radial mononeuropathy is a compressive neuropathy of the radial nerve from prolonged, direct pressure onto the upper medial arm or axilla by an object or surface... This palsy classically involves an individual falling asleep with the arm hanging over a chair or other hard surface, leading to compression within the axilla. An example is immobilization, when an intoxicated individual may not retain the reflexive ability to correct body position while asleep. Another example is an individual falling asleep on the arm of another and consequently compressing that person's nerve, as when a couple sleeps in the same bed. "

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001319
- label: Prolonged exposure to axillary pressure
- parents: [Exposure history (HP:4000110)](https://hpo.jax.org/app/browse/term/HP:4000110)
- pmid: 
- def: History of prolonged compressive pressure in the axillary region.
- comment: Prolonged compressive pressure in the axillary region is a risk factor for compressive radial mononeuropathy.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11042
- synonyms: Prolonged axillary pressure(NONE)
